companion/manifest.json

```
@@ -0,0 +1,26 @@
+{
+  "id": "novastar-splicer",
+  "name": "novastar-splicer",
+  "shortname": "novastar-splicer",
+  "description": "Companion module for the splicer of Novastar",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": "https://github.com/bitfocus/companion-module-novastar-splicer.git",
+  "bugs": "https://github.com/bitfocus/companion-module-novastar-splicer.git/issues",
+  "maintainers": [
+    {
+      "name": "NovaStar",
+      "email": ""
+    }
+  ],
+  "runtime": {
+    "type": "node18",
+    "api": "nodejs-ipc",
+    "apiVersion": "0.0.0",
+    "entrypoint": "../src/main.js"
+  },
+  "legacyIds": [],
```

I believe this should be like this:
```suggestion
  "legacyIds": ["novastar-h-series"],
```

To match and automagically upgrade from the original module.

---

Thanks for your remind